kubernetes/resource_kubernetes_deployment_test.go

```diff
@@ -936,7 +1032,11 @@ func TestAccKubernetesDeployment_regression(t *testing.T) {
 				),
 			},
 			{
-				Config: requiredProviders() + testAccKubernetesDeploymentConfig_regression("kubernetes-local", name),
+				PlanOnly: true,
+				Config:   requiredProviders() + testAccKubernetesDeploymentConfigLocal("kubernetes-local", name, imageName),
+			},
+			{
+				Config: requiredProviders() + testAccKubernetesDeploymentConfigLocal("kubernetes-local", name, imageName),
```

Comment:
The change here is to allow a different configuration for the Released version of the provider vs the locally-compiled version. It should test the new StateUpgraders.